src/robusta/core/reporting/blocks.py

```diff
@@ -467,13 +473,28 @@ class PrometheusBlock(BaseBlock):
     data: PrometheusQueryResult
     metadata: Dict[str, str]
 
-    def __init__(self, data: PrometheusQueryResult, query: str):
+    vertical_lines: Optional[List[PrometheusBlockLineData]]
+    horizontal_lines: Optional[List[PrometheusBlockLineData]]
+    y_axis_type: Optional[ChartValuesFormat]
+    graph_name: Optional[str]
+
+    def __init__(self, data: PrometheusQueryResult, query: str, vertical_lines: Optional[List[PrometheusBlockLineData]],
+                 horizontal_lines: Optional[List[PrometheusBlockLineData]],
+                 y_axis_type: Optional[ChartValuesFormat],
+                 graph_name: Optional[str]):
         """
         :param data: the PrometheusQueryResult generated created from a prometheus query
         :param query: the Prometheus query run
         """
         metadata = {"query-result-version": "1.0", "query": query}
-        super().__init__(data=data, metadata=metadata)
+        super().__init__(data=data, metadata=metadata, vertical_lines=vertical_lines, horizontal_lines=horizontal_lines,
+                         y_axis_type=y_axis_type, graph_name=graph_name)
+
+    def dict(self, *args, **kwargs) -> Dict[str, Any]:
```

Comment:
Why do we need this?

Reply:
ChartValuesFormat is not serializable. so I had to write this helper